File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/Trend.java
package work.socialhub.kmastodon.entity;

import com.google.gson.annotations.SerializedName;

/**
 * @author uakihir0
 */
public class Trend {

    @SerializedName("name")
    private String name;
    @SerializedName("url")
    private String url;
    @SerializedName("history")
    private History[] history;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public History[] getHistory() {
        return history;
    }

    public void setHistory(History[] history) {
        this.history = history;
    }
}

File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/Trend.kt
package work.socialhub.kmastodon.entity

import com.google.gson.annotations.SerializedName

/**
 * @author uakihir0
 */
class Trend {
    @SerializedName("name")
    var name: String? = null

    @SerializedName("url")
    var url: String? = null

    @SerializedName("history")
    var history: Array<History>
}
